app/presenters/tree_node/menu/section.rb

```diff
@@ -0,0 +1,28 @@
+module TreeNode
+  module Menu
+    class Section < TreeNode::Menu::Node
+      set_attribute(:key) { "#{@options[:node_id_prefix]}___tab_#{@object.id}" }
+
+      set_attribute(:title) { _(@object.name) }
+
+      set_attribute(:image) { "100/feature_node" }
```

Comment:
Hehe, yeah, that's the magic of rails :-) which usually only breaks in production.. By making the first access to such an icon take about 2 minutes on a new appliance, became its not in the index and rails tries *really hard* to search for it :-)